mothur_SilvaRef.bat

```diff
@@ -4,17 +4,18 @@
 # execute within project directory
 
 # rename downloaded database
-#system(cp Silva.seed_128 SilvaSeed.fasta)
-#or using the silva reference files from the MiSeq SOP: silva.bacteria.fasta 
+#system(cp Silva.seed_128.align SilvaSeed.fasta)
+#or using the updated silva reference files from the MiSeq SOP: silva.nr_v128.align 
 
 # aligns known sequence to complete database
-# output: E.coliV1V3.align, E.coliV1V3.align.report, E.coliV1V3.flip.accnos file
-align.seqs(fasta=silva/E.coliV1V3.fas, template=silva/silva.bacteria.fasta)
+# move silva.nr_128.align to the silva directory
```

Comment:
This information should be included in the README to describe how the project directory is set up